e2e/cypress/support/db_commands.d.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+/// <reference types="cypress" />
+
+// ***************************************************************
+// Each command should be properly documented using JSDoc.
+// See https://jsdoc.app/index.html for reference.
+// Basic requirements for documentation are the following:
+// - Meaningful description
+// - Each parameter with `@params`
+// - Return value with `@returns`
+// - Example usage with `@example`
+// Custom command should follow naming convention of having `db` prefix, e.g. `dbGetUser`.
+// ***************************************************************
+
+declare namespace Cypress {
+    interface Chainable<Subject = any> {
+
+        /**
+         * Get server config, and assert if it match with the database connection being used by Cypress
```

Comment:
Fix for consistency and small typo:
```suggestion
         * Gets server config, and assert if it matches with the database connection being used by Cypress
```

Reply:
Updated. Thanks @agarciamontoro!